src/App.tsx

```diff
@@ -21,76 +29,32 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(event) => {
+              setItemsPerPage(+event.target.value);
+              setSelectedPage(1);
+            }}
```

Comment:
Move the onClick logic to a separate function. It follows the best practice of separating concerns and keeping your JSX code clean and focused on rendering components rather than defining complex logic directly within it.